feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -97,3 +106,7 @@ func (srv *CentralSessionController) TerminateSession(
 
 	return (&protos.UnimplementedCentralSessionControllerServer{}).TerminateSession(ctx, request)
 }
+
+func (srv *CentralSessionController) Close() {
+	srv.notifHandler.Shutdown()
+}
```

Comment:
this is good. This method may be missing in some of our services.

The reason we do not care much about shutting down is that those services run on docker, so when the service is shut down, the whole container is shut down, so no routines are left behind